### PR TITLE
Copy changes

### PR DESCRIPTION
**Pull request by fordie**
Copy changes as requested in 
https://trello.com/c/BA7BZRTP
and
https://trello.com/c/7JLXIOn4

Emails will need testing. I think I've got all the ones that currently exist, but I'm not 100% sure. Copy for emails is here: https://docs.google.com/a/digital.justice.gov.uk/document/d/1uHdXLCHdZYKAUNJQvDKg_nAPDq_kN-GWuUmW5i3KAXM/edit

### CODE REVIEW

File: app/views/sessions/_login_page.html.haml [hunk 1]
```diff
@@ -11,7 +11,7 @@
 - if feature_enabled?('token_auth')
   %h2.h3.no-line= t('.heading_other')
 
-  %p= t('.body_other')
+  //%p= t('.body_other')
```
Comment (threedaymonk) [inline app/views/sessions/_login_page.html.haml]: Don't comment out code: just delete it.

File: config/locales/en.yml [hunk 2]
```diff
@@ -58,7 +57,7 @@ en:
     memberships:
       member_removed: "Removed %{person} from %{group}"
     tokens:
-      expired_token: "The authentication token has expired and is more than %{time} hours old"
+      expired_token: "The authentication token has expired and is more than %{time} hours only. Request a new link to log in."
```
Comment (threedaymonk) [inline config/locales/en.yml]: "more than N hours only" doesn't make sense!

File: app/views/token_mailer/new_token_email.html.erb [hunk 3]
```diff
@@ -10,35 +10,22 @@
...
         <tr>
           <td width="560" align="left" valign="top" style="font-family:Arial, sans-serif;font-size:16px;font-weight:normal;line-height:30px;color:#434343;padding-bottom:20px;border-collapse:collapse;">
-            Hello,
+            Hello <%= @person.given_name %>,
           </td>
         </tr>
         <tr>
           <td width="560" align="left" valign="top" style="font-family:Arial, sans-serif;font-size:16px;font-weight:normal;line-height:30px;color:#434343;padding-bottom:20px;border-collapse:collapse;">
-            Someone has requested to log into the MOJ People Finder with this email address.
+            To access People Finder, visit <strong><%= distance_of_time_in_words(Token.ttl)%></strong> today. For security reasons, this link will be active for 3 hours only.
```
Comment (threedaymonk) [inline app/views/token_mailer/new_token_email.html.erb]: "visit about 3 hours today"?!

File: app/views/token_mailer/new_token_email.html.erb [hunk 4]
```diff
@@ -10,35 +10,22 @@
...
           <td width="560" align="left" valign="top" style="font-family:Arial, sans-serif;font-size:16px;font-weight:normal;line-height:30px;color:#434343;padding-bottom:40px;border-collapse:collapse;">
-            Please note that the token will expire in <strong><%= distance_of_time_in_words(Token.ttl)%></strong>. Also note that each time a new token is requested all previous tokens will be invalidated.
-          </td>
-        </tr>
-        <tr>
-          <td width="560" align="center" valign="top" style="padding-bottom:40px;border-collapse:collapse;">
-            <table cellpadding="15" cellspacing="0" border="0" align="left" bgcolor="#00823b" style="border-bottom-width:2px;border-bottom-style:solid;border-bottom-color:#003518;border-collapse:collapse;mso-table-lspace:0pt;mso-table-rspace:0pt;">
-              <tbody><tr>
-                <td align="left" valign="middle" bgcolor="#00823b" style="border-collapse:collapse;">
-                  <a href="<%= token_url(@token)%>" target="_blank" class="button" style="font-family:Arial, sans-serif;font-size:24px;font-weight:normal;text-decoration:none;line-height:normal;color:#ffffff;background-color:#00823b;">
```
Comment (threedaymonk) [inline app/views/token_mailer/new_token_email.html.erb]: The one thing we absolutely need in the token email is the token.

**Comment by fordie** [comment]
oops. Thanks Paul. well spotted

**Pull request merged**